dashboard/test/ui/features/SectionActionDropdown.feature

```diff
@@ -0,0 +1,57 @@
+@no_mobile
+@dashboard_db_access
+@pegasus_db_access
+Feature: Using the SectionActionDropdown
```

Comment:
Great new tests!  I suspect we had some UI tests elsewhere that depended on the buttons we're removing with this change.  Have you checked for this?